readme.md

```diff
@@ -39,7 +39,7 @@ URL to normalize.
 
 ##### normalizeProtocol
 
-Type: `boolean`  
+Type: `boolean`
```

Comment:
Undo remove spaces. Use the `.editorconfig` plugin for your text editor to prevent this.